db/seeds.rb

```diff
@@ -24,6 +24,14 @@
   unique_by: :name,
 )
 
+# load Withdraw reasons
```

Comment:
No need to set timestamps. It's automatically done as you know.

`WithdrawalReason.upsert_all(WithdrawalReasons::SEED)`

Reply:
oh cool!